### PR TITLE
Proxy 17- added Unit tests

### PR DESCRIPTION
**Pull request by kabiakou**

### CODE REVIEW

File: js/__tests__/validators/WereDangerousValidator.spec.ts [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+describe('validateWereDangerous function', () => {
+    it('input param is not boolean', async () => {
+        await expect(() => validateWereDangerous('qwertyui')).toThrow(error)
+    })
+})
+
+describe('validateWereDangerous function', () => {
+    it('input param is blank', async () => {
+        await expect(() => validateWereDangerous('   ')).toThrow(error)
+    })
```
Comment (mdziarkach) [inline js/__tests__/validators/WereDangerousValidator.spec.ts:26]: you can put all test cases into one describe

Comment (kabiakou) [reply js/__tests__/validators/WereDangerousValidator.spec.ts:26]: fixed

File: tsconfig.json [hunk 2]
```diff
@@ -7,10 +7,9 @@
         "moduleResolution": "node",
         "rootDir": "./",
         "esModuleInterop": true,
-        // "noImplicitAny": false
```
Comment (mdziarkach) [inline tsconfig.json:10]: why did you commented it? Looks like you don't have `any` types yet

**Review by mdziarkach — APPROVED** [review]

File: tsconfig.json (hunk 2 above)
Comment (kabiakou) [reply tsconfig.json:10]: fixed

**Pull request merged**